Cargo.toml

```diff
@@ -83,3 +83,6 @@ required-features = ["random"]
 name = "test-predicate"
 path = "tests/predicate.rs"
 required-features = ["random"]
+
+[patch.crates-io]
```

Comment:
will fix this patch once https://github.com/FuelLabs/fuel-asm/pull/53 is merged and released.